1-js/07-object-properties/02-property-accessors/article.md

```diff
@@ -94,11 +94,7 @@ alert(user.name); // Alice
 alert(user.surname); // Cooper
 ```
 
-As the result, we have a "virtual" property `fullName`. It is readable and writable, but in fact does not exist.
-
-```smart header="No way to handle `delete`"
-There's no similar method to handle deletion of an accessor property. Only getter/setter methods may exist.
-```
+As the result, we have a "virtual" property `fullName`. It is readable and writable.
```

Comment:
I kind of liked the "in fact does not exist" part. Though that is already implied by "virtual", a bit of emphasis can be helpful. (Though I would have been tempted to rephrase it as "but does not in fact exist"). Without it, "It is readable and writable" seems to just kind of hang there.